src/include/libpmemlog.h

```diff
@@ -83,6 +83,9 @@ void pmemlog_close(PMEMlogpool *plp);
 int pmemlog_check(const char *path);
 size_t pmemlog_nbyte(PMEMlogpool *plp);
 int pmemlog_append(PMEMlogpool *plp, const void *buf, size_t count);
+#ifdef _WIN32
+_When_(return != -1, _Requires_lock_held_(plp->rwlockp))
```

Comment:
This is a public header and the plp structure is private for the library, this cannot stay this way.